README.md

```diff
@@ -96,6 +96,18 @@ Alternatively you can replace your local version with the one on the cluster and
 
 ![editor pull or push](images/editor-pull-push.png)
 
+#### Diff
+
+If a resource exists on the cluster the editor allows you to compare your editor to it. In the toolbar you'll find an action for this sake.   
+
+![editor toolbar diff](images/editor-toolbar-diff.png)
+
+The notifications in the editor also allows you to compare your editor to the resource on the cluster. 
```

Comment:
Not an English expert but because notifications is plural then allows should be allow ?